confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceInputContext.java

```diff
@@ -34,4 +34,13 @@ public interface ConfluenceInputContext
      * @return the properties controlling the behavior of the Confluence input filter stream
      */
     ConfluenceInputProperties getProperties();
+
+    /**
+     * @return the confluence package being filtered
+     * @since 9.26.0
+     */
+    default ConfluenceXMLPackage getConfluencePackage()
+    {
+        return null;
```

Comment:
Not sure it make much sense to return null, it's not going to make it work better. IMO should not implement it and make official that there is an API breakage (the breakage only being that any custom implementation of ConfluenceInputContext, which is a very unlikely event, should implement this new method).